src/context/WalletProvider/KeepKey/setupKeepKeySdk.ts

```diff
@@ -0,0 +1,32 @@
+import { KeepKeySdk } from '@keepkey/keepkey-sdk'
+
+interface PairingInfo {
```

Comment:
does the sdk not export these types? why should the consumer be defining these?

Reply:
updated